src/cgame/default/viewcontrollers/MultiplayerViewController.c

```diff
@@ -31,6 +31,84 @@
 #pragma mark - Actions
 
 /**
+ * @brief ActionFunction for the Quickjoin button.
+ * @description Selects a server based on minumum ping and maximum players with
+ * a bit of lovely random thrown in. Any server that matches the criteria will
+ * be weighted by how much "better" they are by how much lower their ping is and
+ * how many more players there are.
+ */
+static void quickjoinAction(Control *control, const SDL_Event *event, ident sender, ident data) {
+
+	const int16_t max_ping = Clamp(cg_quickjoin_max_ping->integer, 0, 999);
+	const int16_t min_clients = Clamp(cg_quickjoin_min_clients->integer, 0, MAX_CLIENTS);
+
+	GList *server = cgi.Servers();
```

Comment:
This is great, just name this variable "list" and name the `cl_server_info_t * server`.